1주차/[Lv.2]_[1차]_캐쉬/이현준.js

```diff
@@ -0,0 +1,20 @@
+function solution(cacheSize, cities) {
+  const cacheArr = [];
+  const lowercaseCities = cities.map((city) => city.toLowerCase());
```

Comment:
```suggestion
  cities = cities.map((city) => city.toLowerCase());
```
물론 인자를 다루는 것은 권장되지는 않지만 코드를 자세히 보지 않는(?) 환경에서는 위와 같은 식으로 표현해서 메모리를 아끼는 것이(?) 괜찮을 것 같습니다! (당연히 `const`를 사용하는 것이 권장되고 맞습니다!! 😆👏)